src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java9/InputJava9TryWithResources.java

```diff
@@ -20,3 +20,86 @@ public static void main(String[] args) throws Exception {
         try (resource1;resource2) { } finally { }
     }
 }
+
+class TwrForVariable1 implements AutoCloseable  {
```

Comment:
While this compiles, I think it may be better to integrate the examples inside the existing `InputJava9TryWithResources` class rather than adding a separate top-level `TwrForVariable1` class here. @harsh-kukreja, can you refactor?

Reply:
@esilkensen can u resolve this?